### PR TITLE
Implement missing-blocks-guardian and include it into deb pkg

### PR DESCRIPTION
**Pull request by Smorci**
Explain your changes:
* Add missing-blocks-guardian script which is inspired from download-missing-blocks
* Copy script to archive debian package

Explain how you tested your changes:
* local test on a healthy mainnet archive database hosted on AWS by MF
* I am interested in testing it on a broken database
* I am interested in testing it from the debian package

Checklist:

- [x] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Test on broken DB
- [ ] Test from debian package

The script should always report some meaningful message be that usage, error or just info. If other behavior is noticed, please notify me. 

@emberian I included you because this will be included in the [maintenance replayer  package](https://github.com/MinaProtocol/mina/pull/15356/files#diff-d22403ecf1adf48eec7d6bb7e5b7a0753d2af77b872e25cdcb88588a2ebda989) once it is done. 

@dkijania could you help me come up with an idea to test it on a broken DB?

### CODE REVIEW

**Comment by dkijania** [comment]
@Smorci  in order to test script against broken database I propose you to utilize archive-block app and devnet or mainnet precomputed blocks. scenario would look like below:

1. Create empty mainnet database with schema only
2. Using gsutil download precomputed blocks from o1labs bucket (mina_network_block_data) with prefix devnet or mainnet. You can skip some height (precomputed blocks name format is {network-height-state-hash.json} so you can easily avoid particular heights in order to prepare broken db
3. Using archive block app import into database downloaded precomputed blocks
4. Run guardian

**Comment by dkijania** [comment]
I've tested script using mainnet network . Looking good

**Comment by dkijania** [comment]
@Smorci I think we can remove `src/app/rosetta/download-missing-blocks.sh` script which duplicates script you just developed

File: scripts/archive/missing-blocks-guardian.sh [hunk 1]
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
+
+# This script is adapted from https://github.com/MinaProtocol/mina/blob/develop/src/app/rosetta/download-missing-blocks.sh
```
Comment (emberian) [inline scripts/archive/missing-blocks-guardian.sh:3]:
```suggestion
# This script is adapted from https://github.com/MinaProtocol/mina/blob/1.4.1/src/app/rosetta/download-missing-blocks.sh
```

**Review by dkijania — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by Smorci** [comment]
!ci-build-me

**Comment by Smorci** [comment]
!ci-build-me

**Pull request merged**